website/assets/styles/pages/articles/basic-webinar.less

```
@@ -0,0 +1,749 @@
+      display: flex;
+      flex-direction: column;
+      gap: 32px;
+      p {
+        margin-bottom: 0px;
+      }
+
+    }
+  }
+
+  [purpose='article-content'] {
+    word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)